File: day 36/classwork/classwork04.py
#5) უკვე შექმნილ Dictionary-დან, შეცვალე key ისე, რომ value იგივე დარჩეს და დაბეჭდე Dictionary საბოლოო   სახით

dic = {
    "name" : "andria",
    "surname" : "begiashvili",
    "age" : 14
}

dic ["სახელი"] = dic.pop("name")
print(dic)

File: day 60/classwork/classwork04.js
// 5) შექმენი ობიექტი რომელსაც ექნება მეთოდი, ობიექტს ექნება სახელი და ასაკი,
//  ამ მეთოდმა უნდა გააკეთოს "ჩემი სახელია [სახელი] და ვარ [ასაკი]"

const intro = {
    name: "andria",
    age: 14,
    greet(){
        console.log("my name is ", this.name, " i am ", this.age, " years old")
    }
}

intro.greet()